cityofaustin/atd-vz-data
# Adds scooter and motorcycle filters to crash table

## Associated issues
fixes https://github.com/cityofaustin/atd-data-tech/issues/7081

this pr updates the advanced filters on the crashes table to include motor vehicle, motorcycle and scooter rider, and remove motorized conveyance

## Testing
**URL to test:** 
https://deploy-preview-1226--atd-vze-staging.netlify.app/#/crashes

**Steps to test:**
- go to the crashes table and click advanced filters
- try each of the toggles and check one or two records to confirm the records match
- check the scooter rider filter. if there are no records initially, you may need to change an existing unit to type  'micromobility' style 'e-scooter'. change a scooter record to another mode and confirm it no longer appears in the scooter results.
- note the total number of records with no filters applied, then turn on all the filters. the number should be close to the total but missing a few records. we will write more queries to try to capture the missing records in a future pr.

---
#### Ship list
